src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,86 @@
+
+We're making great strides on our journey towards SOC2 certification, striving to meet the highest industry standards for security and privacy. While we're not quite ready to disclose specific milestones, be assured that everything is progressing smoothly. As we continue working diligently towards our target, we promise to keep you informed every step of the way. Our unwavering commitment to deliver secure and private services to our customers and partners remains our foremost priority. Stay tuned for more updates!
+
+## What's next?
+
+We're always working to enhance your experience with FlowForge. Here's how you can stay informed and contribute:
+
+- **Roadmap Overview**: Check out our [Product Roadmap Page](https://flowforge.com/product/roadmap/) to see what we're planning for future updates.
+- **Entire Roadmap**: Visit our [Roadmap on GitHub](https://github.com/orgs/flowforge/projects/5) to follow our progress and contribute your ideas.
+- **Feedback**: We're interested in your thoughts about FlowForge. Your feedback is crucial to us, and we'd love to hear about your experiences with the new features and improvements. Please share your thoughts, suggestions, or report any [issues on GitHub](https://github.com/flowforge/flowforge/issues/new/choose). 
+
+Together, we can make FlowForge better with each release!
+
+## Bug Fixes
+
+...
+
+## Try it out
+
+We're confident you can have self managed FlowForge running locally in under 30 minutes.
+You can install our [local build](https://flowforge.com/docs/install/local/), use [Docker](https://flowforge.com/docs/install/docker/), or [Kubernetes](https://flowforge.com/docs/install/kubernetes/).
+
+If you'd rather use our hosted offering: [Get started for free](https://app.flowforge.com/account/create) on FlowForge Cloud.
+
+## Upgrading FlowForge
+
+[FlowForge Cloud](https://app.flowforge.com) is already running 1.8.
+
+If you installed a previous version of FlowForge and want to upgrade, our documentation provides a
+guide for [upgrading your FlowForge instance](https://flowforge.com/docs/upgrade/).
+
+## Getting help
+
+Please check FlowForge's [documentation](https://flowforge.com/docs/) as the answers to many questions are covered there.
+
+If you hit any problems with the platform please raise an [issue on GitHub](https://github.com/flowforge/flowforge/issues).
+That's also a great place to send us any feedback or feature requests.
+
+You can also get help on [the Node-RED forums](https://discourse.nodered.org/)
+
+As well as in the [forum within our Github project](https://github.com/flowforge/flowforge/discussions)
+
+Chat with us on the `#flowforge` channel on the [Node-RED Slack workspace](https://nodered.org/slack)
+
+You can raise a support ticket by emailing [support@flowforge.com](mailto:support@flowforge.com)
+
+We've also added a live chat widget to our website, you can access it using the icon on the bottom right corner of our website. We'd love to hear from you.
```

Comment:
Can we clean up this block?

I'd say we should link to the docs indeed, than just redirect everything else to community.flowforge.com?

```suggestion
## Getting help

Please check FlowForge's [documentation](https://flowforge.com/docs/) as the answers to many questions are covered there. Additionally you can go the the [community forum](https://community.flowforge.com) if you have
any feedback or feature requests.
```